### PR TITLE
#80 Add warning message when no data collected

### PR DESCRIPTION
**Pull request by RiyaJohn**

### CODE REVIEW

File: wily/commands/build.py [hunk 1]
```diff
@@ -83,6 +83,9 @@ def build(config, archiver, operators):
                     run_operator,
                     [(operator, revision, config) for operator in operators],
                 )
+                for i in range(0, len(operators)):
+                    if i < len(data) and len(data[i]) >= 2 and len(data[i][1]) == 0:
```
Comment (tonybaloney) [inline wily/commands/build.py]: Where is the number 2 from? I guess it has some significance

Comment (RiyaJohn) [reply wily/commands/build.py]: So from what I understand, **data** is a list of tuples, where each tuple is of length **2** and hence checking if len(tuple) >=2. > for safer side as I didn't want to enforce tuple's size to 2

Comment (tonybaloney) [reply wily/commands/build.py]: It stumped me, so it'll stump someone else. Please, could you write it differently so it's more obvious what the code does? If there's no way of doing this, a comment could suffice

Comment (RiyaJohn) [reply wily/commands/build.py]: Tried writing differently with a comment, please review

**Pull request merged**